robot/EDA/resources/locators.py

```diff
@@ -144,7 +144,7 @@
     "eda_tile": "//div[@class='slds-app-launcher__tile-body']//a[contains(text(),'EDA')]/../..",
     "new_account": "//span[@title='New Account']",
     "new_account_next_button": "//button[contains(@class, 'slds-button')]//span[@class=' label bBody' and text()='Next']",
-    "new_account_name": "//input[@class='input uiInput uiInputText uiInput--default uiInput--input']",
+    "new_account_name": "//label/span[text()='Account Name']/following-sibling::span/following::input[1]",
```

Comment:
@dashattuck @kdy618 It looks like we have PR #968 open for this change, so can this be removed from this PR? Otherwise it doesn't make sense to have it in both PRs.

I would prefer if we keep any Robot changes separate from our actual managed package work.

Reply:
I'll put the original back.  Is there a better/different way to back out the change I submitted?